documentation/docs/guidedtour.md

```diff
@@ -12,29 +12,44 @@ The stated instructions assume the use of this application.
 
 * You have installed a Linux system with at least 4 GB memory. **Note:** We have tested our samples on Ubuntu 16.04. On Microsoft Windows, you might face some issues.
 * You have installed the newest version of Docker. See [Docker Community Edition](https://docs.docker.com/install/). **Note:** we have tested on Docker 18.09.6.
-* You have installed Jenkins 2.60.3 or higher. **Recommendation:** We recommend to use the `cx-server` toolkit. See **(Optional) Install the `cx-server` Toolkit for Jenkins**. **Note:** If you use your **own Jenkins installation** you need to care for "Piper" specific configuration. Follow [my own Jenkins installation][guidedtour-my-own-jenkins].
 * Your system has access to [GitHub.com][github].
 
-## (Optional) Install the `cx-server` Toolkit for Jenkins
+## Recommended: Install the Cx Server Life-cycle Management for Jenkins
 
-`cx-server`is a lifecycle management toolkit that provides Docker images with a preconfigured Jenkins and a Nexus-based cache to facilitate the configuration and usage of Jenkins.
+Cx Server is a life-cycle management tool to bootstrap a pre-configured Jenkins instance within minutes.
+All required plugins and shared libraries are included automatically.
+It is based on Docker images provided by project "piper".
 
-To use the toolkit, get the `cx-server` script and its configuration file `server.cfg` by using the following command:
+To get started, initialize Cx Server using this `docker run` command:
 
 ```sh
 docker run -it --rm -u $(id -u):$(id -g) -v "${PWD}":/cx-server/mount/ ppiper/cx-server-companion:latest init-cx-server
 ```
 
-When the files are downloaded into the current directory, launch the Jenkins server by using the following command:
+This will create a few files in your current working directory.
+The shell script `cx-server` and the configuration file `server.cfg` are of special interest.
+
+Now you can start the Jenkins server by using the following command:
 
 ```sh
+chmod +x ./cx-server
 ./cx-server start
 ```
 
-For more information on the Jenkins lifecycle management and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server][devops-docker-images-cxs-guide].
+For more information on Cx Server and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server][devops-docker-images-cxs-guide].
+
+### On your own: Custom Jenkins Setup
+
+If you use your own Jenkins installation you need to care for "Piper" specific configuration.
+This option should only be considered if you know why you need it, otherwise using Cx Server life-cycle management will make your life much easier.
```

Comment:
```suggestion
This option should only be considered if you know why you need it, otherwise using the Cx Server life-cycle management makes your life much easier.
```